src/app/shared/modules/container.module.ts

```diff
@@ -28,6 +28,10 @@ import { DockerfileComponent } from '../../container/dockerfile/dockerfile.compo
 import { DescriptorsComponent } from '../../container/descriptors/descriptors.component';
 import { ParamfilesComponent } from '../../container/paramfiles/paramfiles.component';
 
+/* Bootstrap */
```

Comment:
Move these Bootstrap imports up to the top near the other third party imports.  Also arrange by alphabetical order when possible.